File: pset1/pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_pyramids(int size);

int main(void)
{
    // TODO: Prompt for size: " let’s allow the user to decide just how tall the pyramids should be"
    int size;
    do
    {
        size = get_int("Input the size of the pyramids (1-8): ");
    }
    while (size < 0 || size > 8);
    // "positive integer between, say, 1 and 8, inclusive"
    // Call the function and pass the size
    print_pyramids(size);
}

    // Function to print the pyramids
void print_pyramids(int size)
{
    // Cicle to print size (heigth)
    for(int i = 0; i < size; i++)
    {
        //Cicles to print the width
        // Here we print out the spaces (Depending on the row is the number of spaces)
        for(int space = i; space < size; space++)
        {
             printf(".");
        }
        // Here we just print the character (depending on the row) to build the pyramid
        for(int j = 0; j <= i; j++)
        {
             printf("#");
        }
        printf("\n");
    }
}

File: pset6/pset6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

# prompt the user with get_int
# positive integer between 1 and 8, inclusive
# re-prompt if wrong
while True:
    size = get_int("Height: ")
    if 1 <= size <= 8:
        break

# For each row (the height) of the pyramid
for height in range(size):
    # Print the leading spaces for the left pyramid
    print(" " * (size - height - 1), end="")
    #
    print("#" * (height + 1), end="")
    print("  ", end="")
    print("#" * (height + 1))
